src/main/webapp/app/shared/sidebar/accordion-add-options/accordion-add-options.component.html

```diff
@@ -0,0 +1,22 @@
+@if (sidebarType === 'conversation' && groupKey !== 'favoriteChannels' && groupKey !== 'hiddenChannels') {
```

Comment:
This Component should be more generic. It is very tailored for the conversations.